taxcalc/functions.py

```diff
@@ -289,14 +292,17 @@ def ItemDed(_posagi, e17500, e18400, e18500, e18800, e18900,
 
     # Gross Itemized Deductions
 
-    c21060 = (e20900 + c17000 + (1 - ID_StateLocalTax_HC) * c18300 + c19200 +
-              c19700 + c20500 + c20800 + e21000 + e21010)
+    c21060 = (e20900 + (1 - ID_Medical_HC) * c17000 +
+              (1 - ID_StateLocalTax_HC) * c18300 +
+              (1 - ID_Mortgage_HC) * c19200 +
+              (1 - ID_Casualty_HC) * c19700 +
```

Comment:
Yeah, `c19700` correspond to Line 19, Schedule A, Gifts to Charity section. So I guess here it's a typo.